index.html

```diff
@@ -109,3 +109,12 @@ <h3 class="author">Zeno Rocha</h3>
                     </div>
                 </section>
 {% endblock %}
+ss="work-place">Liferay</span>
```

Comment:
@mateusortiz looks like you accidentally duplicated some markup.
I've fixed it here: https://github.com/JSRocksHQ/harmonic-theme-jsrocks/commit/9529b03f6d9b253b8c81d8b2ded6fe99e49fbb40